interview-questions.md

```diff
@@ -6,50 +6,50 @@ First, without external resources. Challenge yourself to answer from memory as i
 
 Then, research the question to expand on your answer. Even if you feel you have answered the question completely, there is always something more to learn. Write your researched answer in your OWN WORDS.
 
-1. What is JSX? What is one syntax difference between HTML and JSX?
+1. What is JSX? What is one syntax difference between HTML and JSX? 
 
-Your answer:
+Your answer: JSX is a markup language that behaves like html.
 
-Researched answer:
+Researched answer:  JSX is a way to create elements in JS. It utilizes a special syntax that is not available to regular HTML.
 
-1. What is yarn? What file(s) are modified when you run the command yarn in your terminal?
+2. What is yarn? What file(s) are modified when you run the command yarn in your terminal?
 
-Your answer:
+Your answer: Yarn is a package manager. It adds a yarn.lock file and a package.json file.
 
-Researched answer:
+Researched answer: Yarn is a package manager and it is a tool that helps manage and install packages.
 
 3. What is a React component?
 
-Your answer:
+Your answer: A React component is code that is used between a parent and child element. 
 
-Researched answer:
+Researched answer:  A component is a piece of code that is reusable and can be instantiated multiple times. It contains logic and state.
```

Comment:
I would expand some more here. It can certainly include logic and state, but it can also contain a lot more